paddle/pserver/PServerUtil.cpp

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
Please refer to [source file naming convention](https://google.github.io/styleguide/cppguide.html#File_Names) and rename this to be `pserver_util.cpp`.